python/matemenu.c

```diff
@@ -1945,4 +1991,9 @@ DL_EXPORT(void) initmatemenu(void)
 
 	PyModule_AddIntConstant(mod, "SORT_NAME",         MATEMENU_TREE_SORT_NAME);
 	PyModule_AddIntConstant(mod, "SORT_DISPLAY_NAME", MATEMENU_TREE_SORT_DISPLAY_NAME);
+
+#if PY_MAJOR_VERSION < 3
+  mod = NULL;
+#endif
+  return mod;
```

Comment:
This `return` should be there only for Python 3, as with Python 2 the function doesn't return anything...